rnr_control/src/rnr_control.c

```diff
@@ -591,52 +685,115 @@ static void handle_arguments( ps_node_ref const node_ref, int argc, char **argv
                 node_ref,
-                LOGFILE_MODE_OFF,
-                PSYNC_RNR_SESSION_ID_INVALID,
-                rnr_msg );
+                COMMAND_MSG_NAME,
+                &cmd_msg_type );
 
-        // error check
         if( ret != DTC_NONE )
         {
-            psync_log_message(
-                LOG_LEVEL_ERROR,
-                "%s : (%u) -- psync_rnr_fill_logfile_set_mode returned DTC %d",
-                __FILE__,
-                __LINE__,
-                ret );
-
+            psync_log_error( "psync_message_get_type_by_name - ret: %d", ret );
             return;
         }
 
-        // publish command
-        ret = psync_message_publish( node_ref, msg );
+        // allocate command message
+        ret = psync_message_alloc(
+                node_ref,
+                cmd_msg_type,
+                &cmd_msg );
 
-        // error check
         if( ret != DTC_NONE )
         {
-            psync_log_message(
-                LOG_LEVEL_ERROR,
-                "%s : (%u) -- psync_message_publish returned DTC %d",
-                __FILE__,
-                __LINE__,
-                ret );
-
+            psync_log_error( "psync_message_alloc - ret: %d", ret );
             return;
         }
 
+        // construct command to enumerate all dynamic driver interfaces
+        ps_guid dest_guid = PSYNC_GUID_INVALID;
+
+        // all nodes will receive the command message
+        (void) psync_guid_set_node_type( PSYNC_NODE_TYPE_MANAGER, &dest_guid );      
+        
+        // send command to disable the manager runtime nodes, putting the system
+        // in standby mode
+        ret = send_basic_command(
+                node_ref,
+                PSYNC_COMMAND_DISABLE_MANAGER_NODES,
+                dest_guid,
+                (ps_command_msg*) cmd_msg );
+
+        // send command to set the SDF path_to_replay_sdf used by the manager and nodes it manages
+        // to the replay SDF for this logfile session
+       ret = send_command_with_data(
+                node_ref,
+                PSYNC_COMMAND_SET_MANAGER_SDF_PATH,
+                dest_guid,
+                replay_sdf_parameter_value,
+                (ps_command_msg*) cmd_msg );
+       
+               // wait a little
+        (void) psync_sleep_micro( 5000000 );
+
+        // send command to enter replay mode - manager spawns nodes defined in
+        // the replay SDF
+                
+        ret = send_basic_command(
+                node_ref,
+                PSYNC_COMMAND_ENABLE_MANAGER_REPLAY_MODE,
+                dest_guid,
+                (ps_command_msg*) cmd_msg );
+
+        // free the command message
+        ret = psync_message_free(
+                node_ref,
+                &cmd_msg );
+
+        if( ret != DTC_NONE )
+        {
+            psync_log_error( "psync_message_free - ret: %d", ret );
+        }
+
         // wait a little
-        (void) psync_sleep_micro( 500000 );
+        (void) psync_sleep_micro( 5000000 );
```

Comment:
This is 5 seconds, and is way too long. Was empirically derived and required to be this long, or just bumped up for debugging?